app/assets/stylesheets/editor.scss

```diff
@@ -36,6 +36,10 @@
     margin-bottom: 1rem;
   }
 
+  .deposit_header {
+    font-weight: bold;
+  }
```

Comment:
Since this new class only contains a single rule, I wonder if we need it or if we could use the Bootstrap built-in class (`fw-bold`) below instead of the new custom class?